packages/firebase/index.d.ts

```diff
@@ -4479,7 +4479,7 @@ declare namespace firebase.analytics {
      */
     logEvent(
       eventName: 'add_shipping_info',
-      eventParams: {
+      eventParams?: {
```

Comment:
These changes are unrelated to `-compat` and should have a changeset. We should create a separate PR for them.

Reply:
Done: https://github.com/firebase/firebase-js-sdk/pull/4469